modules/core/src/main/java/org/apache/synapse/message/store/MessageStore.java

```diff
@@ -34,39 +35,39 @@
      * Returns a Message Producer for this message store. <br/>
      * @return  A non-null message producer that can produce messages to this message store.
      */
-    MessageProducer getProducer();
+    MessageProducer getProducer() throws SynapseException;
 
     /**
      * Returns a Message Consumer for this message store. <br/>
      * @return A non-null message consumer that can read messages from this message store.<br/>
      */
-    MessageConsumer getConsumer();
+MessageConsumer getConsumer() throws SynapseException;
```

Comment:
Formatting issue

Reply:
Fixed. Older code has a mix of indentations. Indented the whole code. Will loose the change history.